.github/workflows/fix_doctests.yml

```diff
@@ -0,0 +1,27 @@
+name: fix_doctests
+on:
+  pull_request:
+jobs:
+  doctests:
+    name: Fix doctests (Julia ${{ matrix.julia-version }} - ${{ github.event_name }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        julia-version: [1.6]
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - uses: actions/checkout@v1
+      - name: Fix doctests
+	    shell: julia --project=docs/ {0}
+		run: |
+			using Pkg:Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+			using Documenter
+			using ChainRulesTestUtils
+			doctest(ChainRulesTestUtils, fix=true)
```

Comment:
Looks like there might be something off here with tabs/spaces?

Reply:
ah yeah, new computer, I haven't yet fixed all my editors